msteams-platform/bots/how-to/conversations/conversation-messages.md

```diff
@@ -250,13 +260,10 @@ Messages sent between users and bots include internal channel data within the me
 
 ## Get edit message event
 
-When you edit a message in a chat, the bot gets a notification of the edit message event.
+When you edit a message in the group chat or channel, the bot gets a notification of the edit message event.
```

Comment:
update it to "when you edit a message, the bot gets a notification of the edit message **activity**."

Reply:
updated as suggested